### PR TITLE
Translation de de and add missing title prop for message.lighting.global.areas.area

### PR DESCRIPTION
**Pull request by jonilala796**
Added german translation.
Fixed missing title prop for translation

### CODE REVIEW

File: src/components/sections/lighting/subTabs/LightingGlobal.vue [hunk 1]
```diff
@@ -217,7 +217,7 @@ export default {
   <CenteredContainer>
     <ContentContainer style="padding-top: 15px; padding-bottom: 20px">
       <GroupContainer :title="$t('message.lighting.global.areas.title')">
-        <RadioSelection title="Area" group="lighting_global_areas" :options="getAreaOptions()" :selected="this.selected"
+        <RadioSelection :title="$t('message.lighting.global.areas.area')" group="lighting_global_areas" :options="getAreaOptions()" :selected="this.selected"
```
Comment (FrostyCoolSlug) [inline src/components/sections/lighting/subTabs/LightingGlobal.vue:220]: Damnit, knew I'd miss one.. Thanks!

**Review by lm41 — APPROVED** [review]
LGTM

**Comment by FrostyCoolSlug** [comment]
I've merged it in, and it looks great, thanks!

 I'm going to be looking into increasing the size of the 'Mute Behaviour' boxes to better fit, and the Sampler button 'Functions', they seem to commonly overflow..

There's a bit of a question for a one though:

'Threshold'  - Is there a synonym which could possible be used that's shorter? I'm not sure I can fix the thresholds directly without weirdly stretching it for all languages.

**Comment by FrostyCoolSlug** [comment]
Actually, all good.. I've just pushed a commit that lets the boxes stretch slightly for translations:

image

**Pull request merged**